tofhir-engine/src/main/scala/io/tofhir/engine/model/FhirMapping.scala

```diff
@@ -46,6 +46,21 @@ case class FhirMapping(id: String,
       )
     )
   }
+
+  /**
+   * Removes any fields starting with @ from the mapping expression JSON.
+   * @return
+   */
+  def removeAtFields(): FhirMapping = {
```

Comment:
Could you please elaborate on the usage/meaning of these fields i.e. what are they ?

I think we should rename the function name as well if possible.